docs/reference/search-application/apis/put-search-application.asciidoc

```diff
@@ -24,10 +24,38 @@ Also requires <<privileges-list-indices,manage privileges>> on all indices that
 [[put-search-application-path-params]]
 ==== {api-path-parms-title}
 
+
 `create`::
 (Optional, Boolean) If `true`, this request cannot replace or update existing Search Applications.
 Defaults to `false`.
 
+`<body>`::
+(Optional, object)
+Contains parameters for a search application:
++
+.Properties of `<body>` objects
+[%collapsible%open]
+====
+`indices`::
+(Required, array of strings)
+The <<indices>> associated with this search application. All indices need to exist in order to be added to a search application.
+
+`analytics_collection_name`::
+(Optional, string) The name of the behavioral analytics association being used with this search application.
+As of this release, this is used for informational purposes only. For more information see: <<behavioral-analytics-apis
+
+`template`::
+(Optional, object)
+The <<search-template>> associated with this search application. The search application's template is only stored and accessible through the search application.
```

Comment:
```suggestion
The <<search-template,search template>> associated with this search application. The search application's template is only stored and accessible through the search application.
```

When using a link within a sentence, I suggest always providing explicit link text. Several times I've seen the name of a page change, and the new implicit link text breaks the sentence.

I only marked this here, but it would apply throughout this PR.